client/state/themes/selectors/get-is-live-preview-supported.ts

```diff
@@ -125,7 +126,8 @@ export const getIsLivePreviewSupported = ( state, themeId, siteId ) => {
 		 * Disable Live Preview for 3rd party themes, as Block Theme Previews need a theme installed.
 		 * Note that BTP works on Atomic sites if a theme is installed.
 		 */
-		if ( isExternallyManagedTheme( state, themeId ) || isWporgTheme( state, themeId ) ) {
+		// eslint-disable-next-line @typescript-eslint/no-explicit-any
+		if ( isExternallyManagedTheme( state, themeId ) || isWporgTheme( state, themeId as any ) ) {
```

Comment:
I guess we can update the param definition of the `isWporgTheme` function instead of using any here

```diff
-  * @param  {number}  themeId Theme ID
+  * @param  {string}  themeId Theme ID
```

Reply:
I checked other callers and they seemed fine.